src/main/webapp/javascript/RecipeRendering.js

```diff
@@ -45,4 +45,14 @@ createElement = (htmlTag, object = "", tagOptions = {}) => {
         htmlElement.setAttribute(opt, tagOptions[opt]); 
     }
     return htmlElement;
-}
\ No newline at end of file
+}
+
+/**
+ * Takes two objects which represent the recipeName and the blobkey
+ * Returns the reference to the constructed image element. 
+ */
+createImage = (recipeName, blobkey) => {
```

Comment:
Why does this take a recipeName and then not use it?

Reply:
lol I forgot! I was going to make the recipe name the alt text

Reply:
Sounds good!